xls/modules/zstd/frame_header_test.cc

```diff
@@ -58,9 +58,19 @@ enum FrameHeaderStatus : uint8_t {
 };
 
 struct ZstdFrameHeader {
-  absl::Span<const uint8_t> kBuffer;
   ZSTD_frameHeader kHeader;
   size_t kResult;
+
+  absl::Span<const uint8_t> kBuffer() const {
+    return absl::MakeConstSpan(buffer_);
+  }
+
+  ZstdFrameHeader(absl::Span<const uint8_t> buffer, ZSTD_frameHeader& h,
```

Comment:
google style says since it has a constructor it should be a class and not a struct

also passing mutable references is less preferrable here than passing by value -- for any large type (which the frameHeader may be) you can std::move the value

Reply:
Changed `ZstdFrameHeader` to class and dropped passing by reference in favor of passing by value + `std::move`